electron/README.md

```diff
@@ -0,0 +1,113 @@
+# Deploying TensorFlow.js Models in Electron-based Desktop Apps
+
+This is an example that showcases how to use TensorFlow.js
+in cross-platform desktop apps written with
+[electron](https://electronjs.org/). In particular, it uses
+a MobileNetV2 model running in TensorFlow.js to enable the user
+to search for image files on the local filesystem by content
+key words (e.g., "daisy", "chair").
+
+## How to use this example
+
+1. Build and launch the app using:
+
+   ```sh
+   yarn
+   yarn start
+   ```
+
+   If you have a CUDA-enabled GPU on your machine and have set up
```

Comment:
I'd really consider leaving this out - I know this is the most performant option - but shipping desktop applications that require system CUDA is something unreasonable for users. Those of us working on models might have CUDA setup - but most users will not.

Imagine this scenario:
"In order to use my app - please only use Linux or Windows and painfully install this very specific NVIDIA CUDA driver".

When `tfjs-headless-nodegl` launches - this is an option for GPU acceleration w/o expecting users to have a non-default driver.

Reply:
+1